selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/core/client/keycloak/executor/DockerContainerCommandExecutor.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.selenium.core.client.keycloak.executor;
+
+import static java.lang.String.format;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import java.io.IOException;
+import org.eclipse.che.selenium.core.utils.process.ProcessAgent;
+import org.eclipse.che.selenium.core.utils.process.ProcessAgentException;
+
+/** @author Dmytro Nochevnov */
+@Singleton
+public class DockerContainerCommandExecutor implements KeycloakCommandExecutor {
```

Comment:
In my opinion, the current name of the class "DockerContainerCommandExecutor" means that it describes the class which responsible for commands execution in some abstract docker container but not in the particular "Keycloak" container. Maybe it should be renamed to something like "DockerKeycloakCommandExecutor" or "KeycloakCommandExecutorImpl"?

Reply:
You are particularly right, but I took into account the name of package: `keycloak.executor`.